.github/workflows/docs-checks.yml

```diff
@@ -48,11 +48,21 @@ jobs:
           sudo apt-get install -y cmake pandoc
           pip --version
           # python -m pip install --upgrade --user pip
-          pip install -e . --quiet -r requirements/${{ matrix.pkg }}/docs.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+          pip install -e . -r requirements/${{ matrix.pkg }}/docs.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
```

Comment:
Install PyTorch Lightning from source for app docs

Reply:
It's not possible to install dev versions of both PL and app in a single command like this, is it?
```shell
# install both of PL and app from source
PACKAGE_NAME=pytorch,app pip install -e .
```

Reply:
It is with:
```bash
# install both of PL and app from source
PACKAGE_NAME="" pip install -e 
```
https://github.com/Lightning-AI/lightning/blob/7987a1b453150f6c9733db24ab0cdcf11e60dd9d/setup.py#L23

Reply:
I totally missed that option! I'm giving it a try now

Reply:
> ```bash
> # install both of PL and app from source
> PACKAGE_NAME="" pip install -e 
> ```

I tried `PACKAGE_NAME="" pip install -e .` for the app case, but somehow it doesn't install PL at all in CI (on the other hand, locally, I see dev version of PL installed with the same command.):
https://github.com/Lightning-AI/lightning/runs/8113395266?check_suite_focus=true

Any idea...? (any suggestions or commits are welcome...)